### PR TITLE
Added 'header_visible' parameter to allow switching off the header.

### PR DESCRIPTION
**Pull request by ghost**
Some users (like me) may want to hide the header with the site's title.
I added a configuration parameter, `header_visible` (default `true`).

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -47,13 +47,15 @@
     {{ end }}
 
     {{ block "header" . }}
+    {{ if (ne ($.Param "header_visible") false) }}
```
Comment (alanorth) [inline layouts/_default/baseof.html]: I can't remember the difference between the two params syntaxes, but I'd prefer you use the same syntax that I'm using for the description a few lines below:

```
{{ if (ne .Site.Params.header_visible false) }}
```

**Comment by alanorth** [comment]
Sure, I don't mind this change. See my one comment.

Also, next time you should make the changes in a feature branch of your repository, like `no-header` or something, instead of master so that your repository doesn't get out of sync with mine (aka the upstream master) when I make other changes or merge other pull requests.

**Comment by ghost** [comment]
Thanks for the feedback! I made the changes and will use a branch next time.

**Comment by alanorth** [comment]
Hello, I manually cherry-picked your two commits directly to master to avoid merging your branch history. Thanks!

**Pull request closed without merging**